frontend/angular.json

```diff
@@ -136,11 +138,11 @@
   "defaultProject": "o-neko",
   "schematics": {
     "@schematics/angular:component": {
-      "prefix": "app",
-      "styleext": "scss"
+      "prefix": "oneko",
```

Comment:
Does this apply to / override the tslint rules?

If not, we should modify the `tslint.json` to match, e.g.:
```json
"directive-selector": [
  true,
  "attribute",
  ["oneko", "on"],
  "camelCase"
],
"component-selector": [
  true,
  "element",
  ["oneko", "on"],
  "kebab-case"
],
```

Reply:
This change should be applied to both files `angular.json` and `tslint.json`. (See newest commit)